README.markdown

```diff
@@ -458,21 +465,15 @@ cd ../cryptominisat/scripts/fuzz/
 
 Using the Machine Learning System
 -----
-This is experimental but should work relatively well:
-
+This is experimental but should work relatively well. First install required dependencies as per the [Compiling in Linux](#compiling-in-linux) section. To build and install, issue:
 ```
-git checkout clauseID
-sudo apt-get install build-essential cmake git
-sudo apt-get install libzip-dev libboost-program-options-dev libm4ri-dev libsqlite3-dev
-sudo apt-get install graphviz
-sudo apt-get install python3-pip python3-setuptools python3-dev
-sudo apt-get install python3-numpy
 sudo pip3 install --upgrade pip
 sudo pip3 install lit
 sudo pip3 install scikit-learn pandas scipy
 git clone https://github.com/msoos/cryptominisat.git
 cd cryptominisat
 git submodule update --init
+git checkout clauseID
```

Comment:
Do you want me to split this change out into a separate commit? In the current set of docs, the `git checkout` comes before the `git clone`, so it doesn't have the desired effect. Also, the `clauseID` branch seems stale; is this step still needed? This branch is 35 commits ahead, 9223 commits behind master.